NitroxServer/ConsoleCommands/PvpCommand.cs

```diff
@@ -0,0 +1,52 @@
+using System.IO;
+using NitroxModel.DataStructures.GameLogic;
+using NitroxServer.ConsoleCommands.Abstract;
+using NitroxServer.ConsoleCommands.Abstract.Type;
+using NitroxServer.Serialization;
+using NitroxServer.Serialization.World;
+
+namespace NitroxServer.ConsoleCommands;
+
+public class PvpCommand : Command
+{
+    private readonly ServerConfig serverConfig;
+
+    public PvpCommand(ServerConfig serverConfig) : base("pvp", Perms.ADMIN, "Enables/Disables PvP")
+    {
+        AddParameter(new TypeString("state", true, "on/off"));
+
+        this.serverConfig = serverConfig;
+    }
+
+    protected override void Execute(CallArgs args)
+    {
+        string state = args.Get<string>(0).ToLower();
+
+        bool pvpEnabled = false;
+        switch (state)
+        {
+            case "on":
+                pvpEnabled = true;
+                break;
+            case "off":
+                break;
+            default:
+                SendMessage(args.Sender, "Parameter must be \"on\" or \"off\"");
+                return;
+        }
+
+
+        using (serverConfig.Update(Path.Combine(WorldManager.SavesFolderDir, serverConfig.SaveName)))
```

Comment:
Probably not for this PR but I feel like we shouldn't have to specify this whole thing in the command and should just be able to call severconfig.update()

Reply:
Side effect of how bad the DI in server is setup right now, but issue is created for it: #2166